tools/packaging/static-build/shim-v2/build.sh

```diff
@@ -27,6 +27,7 @@ if [ ${arch} = "ppc64le" ]; then
 fi
 
 sudo docker run --rm -i -v "${repo_root_dir}:${repo_root_dir}" \
+	--env GOCACHE=${repo_root_dir} \
```

Comment:
I guess another issue is that we are not caching with github action go mod caches: https://github.com/marketplace/actions/cache